docs/_layouts/default.html

```diff
@@ -1,17 +1,145 @@
 <!doctype html>
 <html lang="en">
     <head>
-        <meta charset="utf-8" />
-        <title>{{ page.title }}</title>
-        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>{{page.title}} (DS{{page.ds}})</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1"/>
         <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+        <link type="text/css" rel="stylesheet" href="/static/ds{{page.ds}}/skin.min.css"/>
+        <link type="text/css" rel="stylesheet" href="/static/ds{{page.ds}}/docs.min.css"/>
+        <!-- critical styles -->
+        <style>
+            @media only screen and (min-width: 601px) {
+                .modules-expander {
+                    display: none;
+                }
+                .site-nav {
+                    display: block;
+                }
+            }
+        </style>
+        <!-- uncomment to load background icons -->
+        <!-- <link type="text/css" rel="stylesheet" href="static/ds{{page.ds}}/skin-base64.min.css"/> -->
     </head>
     <body>
         <header>
-            <h1>{{ page.title }}</h1>
+            <details class="details modules-expander">
+                <summary class="details__summary details_summary--center">
+                    <span class="details__summary-text">modules</span>
+                    <span class="details__icon"></span>
+                </summary>
+                <p class="modules-expander__tip">Use the links below &amp; the back button to navigate</p>
+                <ul class="modules-expander__list">
```

Comment:
Is there a way to maybe put all of these `<li>` in it's own file and then reuse it in both spots in this file. 
It just would be nicer to have to edit a single file rather than do it in multiple spots. Since both lists are the same,.

Reply:
Good call. I'll do that.